packages/react/.storybook/Container.js

```diff
@@ -13,12 +13,6 @@ export default class Container extends Component {
 
   render() {
     const { story } = this.props;
-
-    let bgColor = '#ffffff';
-    if (story().props.context.kind === '[Experimental] UI Shell') {
-      bgColor = '#f3f3f3';
```

Comment:
Just wanted to double-check; Did Shell demo work fine without the custom background color?

Reply:
yes since we renamed the story in https://github.com/carbon-design-system/carbon/commit/73eccb1914d300d480b5b4f2c0cdb4a7a69d40f3#diff-81ee1893617fece3ba05d86230946a1d the background is now always hard coded white. removing these lines sets the background color according to the theme now

and actually I just added 1 more commit to fix another container with hard coded background color and viewport-based height in the UI shell story